macOS (Catalina) xrdp building error

When building under MacOS Catalina I get the following error : 
```
[…]
  CCLD     xrdp-xcon
Making all in chansrv
  CC       chansrv.o
In file included from chansrv.c:32:
In file included from ./devredir.h:24:
In file included from ./irp.h:30:
./chansrv_fuse.h:30:5: error: unknown type name 'off_t'
    off_t           size;              /* Size of file, in bytes.           */
    ^
./chansrv_fuse.h:103:34: error: unknown type name 'off_t'
                                 off_t offset,
                                 ^
2 errors generated.
make[3]: *** [chansrv.o] Error 1
make[2]: *** [all-recursive] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```
Help will be appreciated. Thanks.

The xrdp installation appears to work. After starting xrdp (sudo /usr/local/sbin/xrdp), ticking Screen Sharing for all users, and setting a  VNC password, I can successfully RDP from a Windows (10, build 1909) PC to this macOS xrdp, but ... ONLY using "vnc-any". Neither the "Xorg" for "Xvnc" session managers work. I presume this is as expected with what I have currently installed.

In order to try getting "Xorg" to work, I then moved on to the "xorgxrdp" installation, and this is the new problem where I got stuck : 

```
git clone https://github.com/neutrinolabs/xorgxrdp.git
cd xorgxrdp
./bootstrap
./configure
[...]
checking for pkg-config... /usr/local/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for xorg-server >= 0... no
configure: error: please install xserver-xorg-dev, xorg-x11-server-sdk or xorg-x11-server-devel
```
Is the above "missing" Xorg perhaps because I installed XQuartz (using XQuartz-2.7.11.dmg) instead of for example "xorg-server" from MacPorts ? For me Quartz is working just fine (e.g. for "x3270" which I use a lot).

Any help will be appreciated !

Thanks,

Peter

--- a/sesman/chansrv/chansrv_fuse.c
+++ b/sesman/chansrv/chansrv_fuse.c
@@ -35,7 +35,6 @@ char g_fuse_clipboard_path[256] = ""; /* for clipboard use */
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <time.h>
 
 #include "arch.h"
 #include "chansrv_fuse.h"

--- a/sesman/chansrv/chansrv_fuse.h
+++ b/sesman/chansrv/chansrv_fuse.h
@@ -19,8 +19,10 @@
 #ifndef _CHANSRV_FUSE_H
 #define _CHANSRV_FUSE_H
 
-#include "arch.h"
+#include <sys/types.h>
+#include <time.h>
 
+#include "arch.h"
 #include "ms-erref.h"
 
 /* Used to pass file info back to chansrv_fuse from devredir */

